File: app/src/main/java/com/platdmit/simplecloudmanager/data/database/dao/SizeDao.java
package com.platdmit.simplecloudmanager.data.database.dao;

import com.platdmit.simplecloudmanager.data.database.entity.DbSize;

import java.util.List;

import androidx.room.Dao;
import androidx.room.Delete;
import androidx.room.Insert;
import androidx.room.Query;
import androidx.room.Update;

@Dao
public interface SizeDao  extends BaseDao<DbSize>{
    @Query("SELECT * FROM dbsize")
    List<DbSize> getAll();

    @Query("SELECT * FROM dbsize WHERE netId = :id")
    DbSize getElement(String id);
}

File: app/src/main/java/com/platdmit/simplecloudmanager/data/database/dao/SizeDao.kt
package com.platdmit.simplecloudmanager.data.database.dao

import androidx.room.Dao
import androidx.room.Query
import com.platdmit.simplecloudmanager.data.database.entity.DbSize

@Dao
interface SizeDao : BaseDao<DbSize> {
    @Query("SELECT * FROM dbsize")
    fun getAllElements(): List<DbSize>?

    @Query("SELECT * FROM dbsize WHERE id = :id")
    fun getElement(id: String): DbSize?
}